From: Artem Silenkov <asilenkov@mirantis.com>
Date: Tue, 22 Sep 2015 21:54:44 +0300
Subject: [PATCH] xorriso fails to add files with path longer then 240

joliet-long option added to xorriso command line

Change-Id: I97016cf3e6a554f44d839740ba7d210337435cdc
Related-Bug: #1498619
---
 iso/module.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/iso/module.mk b/iso/module.mk
index 8995378..0073068 100644
--- a/iso/module.mk
+++ b/iso/module.mk
@@ -174,6 +174,7 @@ $(ISO_PATH): $(BUILD_DIR)/iso/isoroot.done
 		-V $(ISO_VOLUME_ID) -p $(ISO_VOLUME_PREP) \
 		-J -R \
 		-graft-points \
+		-joliet-long \
 		-b isolinux/isolinux.bin -no-emul-boot -boot-load-size 4 -boot-info-table \
 		-isohybrid-mbr /usr/lib/syslinux/isohdpfx.bin \
 		-eltorito-alt-boot -e images/efiboot.img -no-emul-boot \
